### PR TITLE
adds heyrutvik to the LLVM ICE-breaker team

### PR DESCRIPTION
**Pull request by heyrutvik**

### CODE REVIEW

**Comment by baloo** [comment]
I believe you did not run the command:
```
cargo run add-person $your_user_name
```
as suggested in the documentation:
https://rust-lang.github.io/rustc-guide/ice-breaker/about.html#joining-an-ice-breaker-group

This should add a file named: `people/heyrutvik.toml` (or maybe you did not `git add`ed it)

**Comment by heyrutvik** [comment]
@baloo PR has that [file](https://github.com/rust-lang/team/pull/163/files#diff-91e0aeda7d3c4446cbdf5f792cddbdc2). Please take a look. Thanks.

**Comment by baloo** [comment]
Damn, sorry for the noise. not sure how I missed it :/

File: teams/icebreakers-llvm.toml [hunk 1]
```diff
@@ -7,4 +7,5 @@ members = [
     "nagisa",
     "nikic",
     "rkruppe",
+    "heyrutvik"
```
Comment (spastorino) [inline teams/icebreakers-llvm.toml]: You can add yourself at the top of the list to keep it sorted.

Comment (heyrutvik) [reply teams/icebreakers-llvm.toml]: Done. Thanks!

**Comment by nikomatsakis** [comment]
oh dear, I tried to merge with the GH UI, but that introduced a merge commit I'd rather not have -- @heyrutvik do you mind rebasing?

**Comment by nikomatsakis** [comment]
Oh, never mind, I can use the "squash and merge" feature. Welcome!

**Pull request merged**